sdk/typespec/typespec_client_core/src/http/response.rs

```diff
@@ -156,12 +143,14 @@ impl<T: Model> Response<T> {
     ///
     /// # Example
     /// ```rust
-    /// # #[derive(serde::Deserialize)]
+    /// # use serde::Deserialize;
+    /// # use typespec_client_core::http::Model;
+    /// # use typespec_derive::Model;
```

Comment:
Shouldn't we make importing the first automatically import the second like serde et. al.? This would make it much easier to consume.

Reply:
Yep, this predates re-exporting the macro from `typespec_client_core`. Fixed now.

Reply:
Ah, I remember. This needs to be here because we don't build doctests with the `derive` feature. Here, it's hidden from the docs. I can change the `deserialize_body_into` doc comment to hide that derive and "fake" using the `derive` feature.